app/assets/stylesheets/modules/_landing.scss

```diff
@@ -0,0 +1,41 @@
+.landing {
+  .banner {
```

Comment:
This is more for namespacing of the CSS. For each of the landing and sponsors page, the main content are wrapped inside this parent `div`. This definitely prevents styles from "leaking" out to other pages/components. I'm cool with making it `.landing-banner` instead of `.banner` also but I think they should all be wrapped inside `.landing`.

I'm aware that it's not a good idea to nest so many levels; There won't be too much nesting. The most is two to three.